packages/features/ee/workflows/lib/getOptions.ts

```diff
@@ -19,7 +24,7 @@ export function getWorkflowActionOptions(t: TFunction, isOrgsPlan?: boolean) {
     return {
       label: actionString.charAt(0).toUpperCase() + actionString.slice(1),
       value: action,
-      needsCredits: !isOrgsPlan && isSMSOrWhatsappAction(action),
+      needsCredits: (!isOrgsPlan && isSMSOrWhatsappAction(action)) || isCalAIAction(action),
```

Comment:
We are using credit based system to deduct credits after the call has ended

Reply:
Do you also have to pay for cal.ai on the orgs plan?

Reply:
Yes, based on usage